chains/mainnet/noble.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
+  chainID: 'noble-1',
+  lcd: 'https://noble-api.polkachu.com',
+  gasAdjustment: 1.75,
+  gasPrices: { uosmo: 0.025 }, // <-- update this
+  prefix: 'noble',
+  coinType: '118',
+  baseAsset: 'uosmo', // <-- update this
```

Comment:
Technically this is `ustake`. It should be noted that this asset is only used and is only available to selected validators.